test/parallel/test-assert-calltracker-calls.js

```diff
@@ -78,3 +78,17 @@ assert.throws(
   callsNoop();
   tracker.verify();
 }
+
+{
+  function func() {}
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(func.length, callsfunc.length);
+}
+
+{
+  function func(a, b, c = 2) {}
+  const tracker = new assert.CallTracker();
+  const callsfunc = tracker.calls(func);
+  assert.strictEqual(func.length, callsfunc.length);
+}
```

Comment:
Let's add a few edge case tests as well:
```suggestion
}

{
  function func(a, b, c = 2) {}
  delete func.length;
  const tracker = new assert.CallTracker();
  const callsfunc = tracker.calls(func);
  assert.strictEqual(Object.hasOwn(callsfunc, 'length'), false);
}

{
  function func(a, b, c = 2) {}
  Object.defineProperty(func, 'length', { get: common.mustNotCall() });
  const tracker = new assert.CallTracker();
  const callsfunc = tracker.calls(func);
  assert.strictEqual(Object.hasOwn(callsfunc, 'length'), true);
}
```